.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: native-metrics CI
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 14.x]
```

Comment:
Thanks for adding 14. This one is an exception cause a customer already helped with a PR to get it working with 14 (and so far haven't heard of issues).